backend/src/zimfarm_backend/db/requested_task.py

```diff
@@ -287,11 +291,11 @@ def compute_task_eta(session: OrmSession, task: Task) -> dict[str, Any]:
 
 
 class RunningTask(BaseModel):
-    config: dict[str, Any]
+    config: ExpandedScheduleConfigSchema
     schedule_name: str
     timestamp: dict[str, Any]
```

Comment:
We can leave it for another PR, but I think this typing is wrong (was here before you came): this is a list of events, supposed to be ordered by order of arrival, and under extreme case (I don't recall exactly, but it is linked to the cancellation of a task while it was just starting on the worker) we can have multiple time the same event (key in the dictionary) and it is a bit sad we loose this info. I let you decide if you want to fix this now or open a distinct issue to fix it a bit later.

Reply:
I will create a new issue to model it properly. I was under the the impression it came from the Task config here https://github.com/openzim/zimfarm/blob/e1cbd9211113575b12425bdfe81ce4dc70024f25/dispatcher/backend/src/utils/scheduling.py#L367-L381

Will craft a better model once I explore the db model